packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -396,26 +397,70 @@ const transform = initialize(SERVICES, EVENTS, FUZZY_EVENTS);
 module.exports = receiveEvent;
 
 function receiveEvent(event, request, data) {
-  if (amplitude.disabled || !event || !request || !data) {
+  if (!event || !request || !data) {
     return;
   }
 
-  const userAgent = ua.parse(request.headers['user-agent']);
+  if (amplitude.rawEvents) {
```

Comment:
I would expect rawEvents to only work if amplitude is enabled, won't this work if disabled and rawEvents is true?

Reply:
I was think keeping them independent so that some day we'll keep the raw and stop the transformed events.

But what you are saying also make sense.  If amplitude as a feature is off, then raw events shouldn't be logged either.

I'll make an update.

Reply:
Updated so that amplitude.disabled is back to en/disable the feature overall.